### PR TITLE
Fixed check telemetry api

### PR DESCRIPTION
**Pull request by manoj22-k**

### CODE REVIEW

File: src/mtool/api/rest/app.py [hunk 1]
```diff
@@ -2063,8 +2063,7 @@ def set_telemetry_config():
         return make_response('Could not configure Telemetry URL'+str(e), 500)
 
 @app.route('/api/v1/checktelemetry', methods=['GET'])
-@token_required
-def check_telemetry(current_user):
+def check_telemetry():
```
Comment (aswinssir) [inline src/mtool/api/rest/app.py:2066]: It is better to add token in the UI, than removing the user check in API

Comment (manoj22-k) [reply src/mtool/api/rest/app.py]: Thanks, I have updated.

**Review by vishal-shakya-ssir — APPROVED** [review]
LGTM

**Review by aswinssir — APPROVED** [review]
LGTM

**Pull request merged**